docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md

```diff
@@ -79,7 +90,18 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
 
 ### Step 2: Configure integration
 
-In this step the user needs to provide the path to the mongo db log file configured as part of above steps. Typically the logs are located at the location: `/var/log/mongodb/mongodb.log`.
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, you will configure the yaml file required for MongoDB collection.
+
+Below are the inputs required:
+
+- **`Endpoint (no default)`**. The hostname and port of the MognoDB instance, separated by a colon. (For example: `localhost:27017`.)
+- **`Logs Path`**. Enter the path to the log file for your MognoDB instance.
+- **`username (optional)`**. Enter the MognoDB username.
+- **`password (optional)`**. Enter the MognoDB password.
+
+User needs to provide the path to the mongo db log file configured as part of above steps. Typically the logs are located at the location: `/var/log/mongodb/mongodb.log`.
 
 You can add any custom fields which you want to tag along with the data ingested in Sumo. Click on the **Download YAML File** button to get the yaml file.
```

Comment:
We also need to mention if authentication for mongo is configured then they need to download env file. Please check the content in any of the other app docs.

Reply:
fixed it.

Reply:
in step3 after this there should be steps to include env file under linux tab. Please refer mysql doc for it.

Reply:
Ok